listenbrainz/webserver/views/index.py

```diff
@@ -155,6 +155,7 @@ def recent_listens():
         "mode": "recent",
         "spotify": spotify_user,
         "api_url": current_app.config["API_URL"],
+        "sentry_dsn": current_app.config["LOG_SENTRY"]["dsn"]
```

Comment:
We need to add this to all pages, so let's use a flask context processor. We already have one for some static files processing, so we should make it more generic: https://github.com/metabrainz/listenbrainz-server/blob/f6b02d9c63/listenbrainz/webserver/__init__.py#L154

Reply:
ah, I realised that these are set in props instead of the page template variables. Any reason to do it here instead? I guess in the js we don't have the template processor, so we can't substitute the value in, is that right?
Perhaps we could render a global app props in addition to local page props.

Reply:
As far as I know, we don't have the template processor in the react side. The props are passed like [this](https://github.com/metabrainz/listenbrainz-server/blob/afb3d43aeaf916aaca059c758571498e0893d497/listenbrainz/webserver/templates/base-react.html#L10) to the React. We could perhaps create a `react-global-props` here and pass some things like the dsn (maybe the api url as well) using the context processor. In the react side, we then use that variable instead of `reactProps`.

Reply:
100% agree with the need for global props. Might be better to refactor that in another PR though.